diff -u -r site-packages.old/dipy/reconst/shm.py site-packages/dipy/reconst/shm.py
--- site-packages.old/dipy/reconst/shm.py	2017-04-26 10:25:38.299779621 +0200
+++ site-packages/dipy/reconst/shm.py	2017-04-26 10:25:06.684762593 +0200
@@ -362,7 +362,7 @@
     n_range = np.arange(0, sh_order + 1, 2, dtype=int)
     n_list = np.repeat(n_range, n_range * 2 + 1)
 
-    ncoef = (sh_order + 2) * (sh_order + 1) / 2
+    ncoef = int((sh_order + 2) * (sh_order + 1) / 2)
     offset = 0
     m_list = empty(ncoef, 'int')
     for ii in n_range:
